packages/@uppy/remote-sources/README.md

```diff
@@ -0,0 +1,35 @@
+# @uppy/remote-sources
+
+<img src="https://uppy.io/images/logos/uppy-dog-head-arrow.svg" width="120" alt="Uppy logo: a superman puppy in a pink suit" align="right">
+
+<a href="https://www.npmjs.com/package/@uppy/remote-sources"><img src="https://img.shields.io/npm/v/@uppy/compressor.svg?style=flat-square"></a> <img src="https://github.com/transloadit/uppy/workflows/Tests/badge.svg" alt="CI status for Uppy tests"> <img src="https://github.com/transloadit/uppy/workflows/Companion/badge.svg" alt="CI status for Companion tests"> <img src="https://github.com/transloadit/uppy/workflows/End-to-end%20tests/badge.svg" alt="CI status for browser tests">
+
+## Example
+
+```js
+import Uppy from '@uppy/core'
+import RemoteSources from '@uppy/remote-sources'
+
+const uppy = new Uppy()
+uppy.use(RemoteSources, {
+  companionUrl: 'https://your-companion-url',
+})
+```
+
+## Installation
+
+```bash
+npm install @uppy/remote-sources
+```
+
+We recommend installing from yarn or npm, and then using a module bundler such as [Parcel](https://parceljs.org/), [Vite](https://vitejs.dev/) or [Webpack](https://webpack.js.org/).
```

Comment:
I thought we decided to stop recommending bundler she by name in READMEs

Reply:
Yes, I wasn't sure if that PR was gonna do that, wanted to avoid conflict :) Will remove!

Reply:
Yes, I wasn't sure if that PR was gonna do that, wanted to avoid conflict :) Will remove!